platform/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Thu Nov 05 15:43:10 CST 2015
+#Fri Jan 15 17:23:43 CET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
-distributionSha256Sum=a88db9c2f104defdaa8011c58cf6cda6c114298ae3695ecfb8beb30da3a903cb
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
```

Comment:
We need to bring the distributinoSha256Sun property over here too.  Sadly this is always a hand copy thing.